twilio/twilio-video-app-android
# VIDEO-4258: Fix issue related to remove the audio and video tracks when the app is killed.(#173)

## Description

Fix Issue to kill the app after removing the app from recent apps. The audio still streams after killing the app.

## Breakdown

- Changed to public, methods `removeCameraTrack` and `removeAudioTrack` in order to be called in the `RoomManager.kt`

- In the `RoomViewModel.kt`, run the disconnect process the ViewModel is cleared.

- Added binder to the `VideoService.kt`, in order to bind it to the app properly.

## Validation

- Open the app in two devices.
- Join both devices to the same room.
- Make sure, the audio and video stream correctly in both directions.
- Now, in one of the devices, send the app to background. 
- Check if the audio still streams (Should be)
- Now, open recent apps, close/kill the app.
- Check if the audio still streams (Should NOT be)

## Additional Notes

When the app is killed, the video and audio tracks, should also be removed from the second device (RemoteTracks)

**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.
